### DISCUSSION TITLE
Can't pay for 2 invoices

### DISCUSSION THREAD

**Discussion by Eiraspray** [swellstores/community#401] [Help] [answered]
The modal window about payment is appearing constantly, I've updated payment info but there is an error for 2 invoices ($1.12 and $0.48): 

<img width="1007" alt="image" src="https://github.com/swellstores/community/assets/147026523/df7725ca-6f32-4959-85e9-215f5bea6fa2">

<img width="900" alt="image" src="https://github.com/swellstores/community/assets/147026523/5469c915-9738-4298-a3e3-e76968b04d9c">

**Comment by steven-k1** [comment, marked as answer]
@Eiraspray For invoice #106531, this did not process due to the amount being under $0.50 Swell uses Stripe for our internal processing of payments, and Stripe does not allow payments under $0.50 to be processed. We are working on a fix for this, but for the time being, I have gone ahead and applied a credit for the $0.48, and the invoice is now processed.

For invoice #115872, I can see that the default card on your account was not set for this particular invoice, and this is the reason for the error. I set the default card for the invoice, and it processed through correctly.

You should be all set now, but please let me know if you have any continued issues.
